src/grt/src/fastroute/src/FastRoute.cpp

```diff
@@ -544,6 +548,30 @@ int FastRouteCore::getEdgeCapacity(long x1,
   return cap;
 }
 
+int FastRouteCore::getEdgeCapacity(FrNet* net,
+                                   int x1,
+                                   int y1,
+                                   int x2,
+                                   int y2)
+{
+  int cap = 0;
+
+  // get 2D edge capacity respecting layer restrictions
+  for (int l = net->minLayer; l <= net->maxLayer; l++) {
+    if (y1 == y2) { // horizontal edge
```

Comment:
It looks like at all the call sites you already know whether you want h or v capacity.  I would just take a bool h/v arg instead of x[12]/y[12]